test/parallel/test-stream-preprocess.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const fs = require('fs');
+const path = require('path');
+const rl = require('readline');
+
+const BOM = '\uFEFF';
+
+// get the data by a non-stream way to compare with the streamed data
+const modelData = fs.readFileSync(
+  path.join(__dirname, '..', 'fixtures', 'file-to-read-without-bom.txt'), 'utf8'
```

Comment:
`path.join(common.fixturesDir, 'file-to-read-without-bom.txt')` (and below where appropriate)?